nemo_skills/dataset/mmlu-pro/__init__.py

```diff
@@ -15,12 +15,9 @@
 
 # settings that define how evaluation should be done by default (all can be changed from cmdline)
 
-PROMPT_CONFIG = 'generic/mmlu-pro-zero-shot'
-DATASET_GROUP = 'multichoice'
-METRICS_TYPE = 'multichoice'  
-DEFAULT_EVAL_ARGS = (
-    "++eval_type=multichoice "
-)
-
+PROMPT_CONFIG = "generic/general-boxed"
+DATASET_GROUP = "multichoice"
+METRICS_TYPE = "multichoice"
+DEFAULT_EVAL_ARGS = "++eval_type=math"
```

Comment:
I think we should have a different eval type here as going through math means running things in sandbox and using sympy for complex evaluation of expressions which feels unneded here?